graphql/ui_plugin_build_baron.go

```diff
@@ -91,23 +81,35 @@ func BbFileTicket(context context.Context, taskId string, execution int) (bool,
 	return taskNotFound, nil
 }
 
-func IsWebhookConfigured(t *task.Task) bool {
+func IsWebhookConfigured(project string) (evergreen.WebHook, bool) {
 	var webHook evergreen.WebHook
 	flags, err := evergreen.GetServiceFlags()
 	if err != nil {
-		return false
+		return evergreen.WebHook{}, false
 	}
 	if flags.PluginAdminPageDisabled {
-		projectRef, err := model.FindOneProjectRef(t.Project)
+		parserProject, err := model.ParserProjectFindOneById(project)
```

Comment:
Parser projects are actually created for every version, not for every project. What you'll want to do instead is look up the most recent valid version for the project (https://github.com/evergreen-ci/evergreen/blob/65cdf890e49b342699e7e56e331a4917b4ae0773/model/version_db.go#L68) and then use that version ID to look up the parser project.